src/LibraryManager.Vsix/Commands/CleanCommand.cs

```diff
@@ -78,6 +78,8 @@ public static void Initialize(Package package, OleMenuCommandService commandServ
 
         private async Task ExecuteAsync()
         {
+            Telemetry.TrackUserTask("CleanCommand");
```

Comment:
Clean-Command

Reply:
In this case, to follow the {verb}-{noun} guideline, I have to rename it to "Execute-CleanCommand"